pkg/apis/v1/nodepool_status.go

```diff
@@ -27,13 +27,18 @@ const (
 	ConditionTypeValidationSucceeded = "ValidationSucceeded"
 	// ConditionTypeNodeClassReady = "NodeClassReady" condition indicates that underlying nodeClass was resolved and is reporting as Ready
 	ConditionTypeNodeClassReady = "NodeClassReady"
+	// TODO
+	ConditionTypeDegraded = "Degraded"
```

Comment:
Do you have thoughts around how you are going to track the last failed launch and then extend the amount of time before we retry out?

Reply:
For this, I was thinking we'd use the `lastTransitionTime` for when `Degraded == true`.